### DISCUSSION TITLE
Can LLM use other LLM as a tool?

### DISCUSSION THREAD

**Discussion by sunsetsobserver** [langflow-ai/langflow#709] [Q&A]
Hello, 

Can LLMs be tools for other LLM agents? 

For instance a GPT-2 model from HuggingFace would be a tool that OpenAI GPT-4 can use if user asks specifically to ask something to GPT2?

I tried to set this up (see picture attached) but after asking any question in a chat window I am getting:
"Error: Saving not supported for this chain type."

I added a placeholder python tool for GPT2 agent as Langflow stated adding tool is required.

I would appreciate any help! Thank you!

<img width="970" alt="LMM as tool for other LLM" src="https://github.com/logspace-ai/langflow/assets/63563505/ca77a1bd-7de1-4d05-b82d-c38ef109a8ca">

**Comment by DoubleLazyZ** [comment]
@sunsetsobserver I would also like to ask if there are any success stories that anyone can share?

**Comment by carlosrcoelho** [comment]
Hi! :wave:
We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.
[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)
Thank you for your understanding.
